templates/elementary-OS-app/README.md.in

```diff
@@ -0,0 +1,20 @@
+# Hello World
```

Comment:
You should use a variable here, like `${PROJECT_NAME}`. See the other templates in Valdo for how this is done.